### PR TITLE
Tech/local registry

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: it can be hard to test reproductions outside of the monorepo

## What I did
I added a bootstrap command that allows you to run a local npm registry which you can then use to install local storybook packages outside of the monorepo.

It's kinda like npm link, but with less hair pulling.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -67,7 +69,7 @@ Before the tests are run, the project must be bootstrapped with core. You can ac
 This option executes tests from `<rootdir>/examples/official-storybook`
 In order for the image snapshots to be correctly generated, you must have a static build of the storybook up-to-date :
 
-```javascript
+```js
```
Comment (shilman) [inline CONTRIBUTING.md]: why is this `js`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -95,68 +101,93 @@ In that case, please check the git diff before committing to make sure it only c
...
 
 **NOTE**: If your issue involves a webpack config, create-react-app will prevent you from modifying the _app's_ webpack config, however, you can still modify storybook's to mirror your app's version of the storybook. Alternatively, use `yarn eject` in the CRA app to get a modifiable webpack config.
 
+#### Outside the monorepo
+
+Sometimes your storybook is deeply ingrained in your own setup and it's hard to create a minimal viable reproduction somewhere else.
+
+Inside the storybook repo we have a script that allows you to test the packages inside this repo in your own seperate project.
+
+Though you can use `npm link` on all packages, this would be work, but npm linking is cumbersome.
```
Comment (shilman) [inline CONTRIBUTING.md]:
```
You can use `npm link` on all packages, but npm linking is cumbersome and has subtle differences from what happens in a registry-based installation.
```

File: addons/a11y/package.json [hunk 3]
```diff
@@ -26,12 +26,12 @@
     "prepare": "node ../../scripts/prepare.js"
   },
   "dependencies": {
-    "@storybook/addons": "5.1.0-alpha.33",
-    "@storybook/api": "5.1.0-alpha.33",
-    "@storybook/client-logger": "5.1.0-alpha.33",
-    "@storybook/components": "5.1.0-alpha.33",
-    "@storybook/core-events": "5.1.0-alpha.33",
-    "@storybook/theming": "5.1.0-alpha.33",
+    "@storybook/addons": "^5.1.0-alpha.33",
```
Comment (shilman) [inline addons/a11y/package.json]: Why all these changes?

**Comment by ndelangen** [comment]
[](http://www.youtube.com/watch?v=Uc47Np0VKyU "")

**Pull request merged**